File: Array/JS/commonElement.js
function commonElement(array1, array2) {
    for (let i = 0; i < array1.length; i++) {
        for (let j = 0; j < array2.length; j++) {
            if (array1[i] == array2[j]) {
                return true;
            }            
        }        
    }
    return false
}

list = [1, 2, 3, 4, 5]
list2 = [8, 7, 5, 6]

console.log(commonElement(list, list2));

File: Array/Python/commonElement.py
def commonElement(arr1, arr2) :
    for el in arr1:
        for me in arr2:
            if el == me:
                return True
    else:
        return False

list = [1, 2, 3, 4, 5]
list2 = [8, 7, 5, 6]

print(commonElement(list, list2))